src/benchmark/cases/mergesort.js

```diff
@@ -0,0 +1,108 @@
+function copyArray(destination, source, includingStartIndex, excludingEndIndex) {
```

Comment:
Maybe rename the `includingStartIndex` and `excludingEndIndex` parameters to `fromIndex` and `toIndex`